notebooks/apify_haystack_rag.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>Using getpass is more user friendly here </p><p> </p>

<pre class="ql-syntax" spellcheck="false">import os
from getpass import getpass

os.environ["APIFY_API_TOKEN"] = getpass("Enter YOUR APIFY_API_TOKEN")
os.environ["OPENAI_API_KEY"] = getpass("Enter YOUR OPENAI_API_KEY")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/105/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Cool, thanks! I was not aware about the `getpass`